tests/unit/tf/horovod/test_embedding.py

```diff
@@ -8,6 +10,7 @@
 
 
 @pytest.mark.skipif(not HAS_GPU, reason="No GPU available")
+@pytest.mark.skipif(importlib.util.find_spec("sparse_operation_kit") is None, reason="needs sok")
```

Comment:
There's only one test class here so this works. However, if this whole test file should be skipped (even if more tests were added in future), then perhaps a top-level `pytest.importorskip("sparse_operation_kit", reason="needs SOK")` would match the intent here too?